circleci/images/citusupgradetester/files/etc/requirements.txt

```diff
@@ -1,9 +1,10 @@
-# generated from Citus's Pipfile.lock (in src/test/regress) as of #6710
+# generated from Citus's Pipfile.lock (in src/test/regress) as of #6720
```

Comment:
It feels weird that we manually edit a generated file. I guess one day we will forget to update this line with the latest PR. However, until that day comes, it is nice to have this.

Reply:
Yeah I don't like it either. But I can't think of a better way either at the moment.